src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Api/Controllers/ComponentDetailsController.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Bit.Client.Web.BlazorUI.Playground.Shared.DTOs;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Client.Web.BlazorUI.Playground.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ComponentDetailsController : ControllerBase
+    {
+        public ComponentDetailsController()
+        {
+            BitComponentAssembly = Assembly.GetAssembly(typeof(BitToggleButton));
+        }
+
+        protected Assembly BitComponentAssembly { get;}
+
+        [HttpGet("GetProperties")]
```

Comment:
use get-properties instead of GetProperties?
what do you think @msynk

Reply:
Yes why not, I'll do it

Reply:
why should this method have a distinct name?
are we going to add other get methods in this controller?

Reply:
Yes, we have another get action for getting component methods detail

Reply:
yeah, i saw it and changed to comment